src/main/flight/pid.h

```diff
@@ -148,6 +148,7 @@ typedef struct pidProfile_s {
     uint8_t abs_control_gain;               // How strongly should the absolute accumulated error be corrected for
     uint8_t abs_control_limit;              // Limit to the correction
     uint8_t abs_control_error_limit;        // Limit to the accumulated error
+    uint8_t abs_control_cutoff;             // Cutoff frequency for path estimation in abs control
```

Comment:
The version of the `pidProfiles` parameter group needs to be increased for this (appending at the end won't help, since it's an array type parameter group.

Reply:
done